src/app/services/bucket/qarnotBucketService.py

```diff
@@ -0,0 +1,21 @@
+from qarnot import connection, bucket
+
+from app.services.bucket.baseBucketService import BaseBucketService
+from app import config as cfg
+
+
+class QarnotBucketService(BaseBucketService):
+
+    @staticmethod
+    def connect_to_bucket(bucket_name: str):
+        conn = connection.Connection(client_token=cfg.QARNOT_TOKEN)
+        return bucket.Bucket(conn, bucket_name)
+
+    async def upload_file(self, bucket_name: str, bucket_key: str, file_binary: bin):
+        return await self.connect_to_bucket(bucket_name).add_file(bucket_name, file_binary, bucket_key)
```

Comment:
connection to the bucket should be done in the constructor in my opinion. And then we can use all qarnot python methods afterwards

Reply:
(that might mean changing the BaseBuckerService, but I believe it's necessary)

Reply:
Could be a good idea but isn't there a risk that the connection closes at some point? 
I don't how it works for Qarnot.

Reply:
Worst case scenario, we change it later with a `try except` but reconnecting all the time could throw an internal security warning for the Qarnot account as well

Reply:
Yes I wanted to do it at the beginning but the issue is that if we establish the connection in the constructor we would need a valid token (the constructor is called and not patched). Until now, I am not using a token because the methods are all patched.
What do you think ?

Reply:
Maybe i could store the connection in an attribute (using the static method I've created) and create the connection object only if it does not already exist. I could try this

Reply:
We can also validate this PR, and open another to refactor this if this becomes cumbersome!
But I agree with your 2nd suggestion :ok_hand: That would be great!